tests/pytorch/test_pytorch_model_export.py

```diff
@@ -738,6 +760,33 @@ def test_load_model_succeeds_when_data_is_model_file_instead_of_directory(
     )
 
 
+@pytest.mark.large
+@pytest.mark.parametrize("scripted_model", [True, False])
+def test_log_state_dict(sequential_model, model_path, data):
+    artifact_path = "pytorch_model"
+    with mlflow.start_run():
+        mlflow.pytorch.log_state_dict(
+            state_dict=sequential_model.state_dict(), artifact_path=artifact_path
+        )
+        model_path = mlflow.get_artifact_uri(artifact_path)
+
+    model_class = nn.Sequential(nn.Linear(4, 3), nn.ReLU(), nn.Linear(3, 1),)
```

Comment:
Can we define a util function like below and and reuse it in the `sequential_model` fixture and here?

```python
def get_sequential_model():
    return nn.Sequential(nn.Linear(4, 3), nn.ReLU(), nn.Linear(3, 1),)
```

Reply:
and `nn.Sequential(nn.Linear(4, 3), nn.ReLU(), nn.Linear(3, 1),)` is not a class but a model. Can you we rename the varible name `model_class`?

Reply:
created `get_sequential_model` and renamed `model_class` to `model`